src/libs/http/client.ts

```
@@ -0,0 +1,126 @@
+      })
+      .catch((error) => {
+        return Promise.reject(error)
+      })
+      return {
+        status: result.status,
+        body: result.data,
+        headers: new Headers(flattenAxiosConfigHeaders(method, result.headers)),
+      }
+    }
+    catch (e: unknown) {
+      throw e
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)